README.md

```diff
@@ -47,21 +46,20 @@ In this case, the output will be:
 {'00': 513, '11': 511}
 ```
 
-A script is available [here](examples/python/ibmq/hello_quantum.py), where we also show how to
-run the same program on a real quantum computer via IBMQ.
+A script is available [here](examples/python/ibmq/hello_quantum.py), where we also show how to run the same program on a real quantum computer via IBM Quantum.
```

Comment:
We're probably going to remove the ibmq examples directory so we can probably just delete this line

Reply:
done in 9296ac8610054488a2858d2bf508d51778a7e26a